app/Http/Controllers/UserController.php

```diff
@@ -17,4 +20,25 @@ public function delete(User $user)
         $user->delete();
         return redirect()->action('UserController@index')->with('Success', 'The user is deleted successfully.');
     }
+
+    public function edit(){
+        return view('profileEdit', ['user' => Auth::user()]);
+    }
+
+    public function store(ProfileRequest $request)
+    {
+        if ($request->avatar != NULL) {
+            $avatar = $request->avatar->storeAs('public/profile_images', Auth::id() . '.jpg');
+        } else {
+            $avatar = "";
```

Comment:
you can use shorthand for this condition.

Reply:
Okay.